### PR TITLE
New Inspector Integration

### PR DESCRIPTION
**Pull request by custompointofview**

### CODE REVIEW

File: robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py [hunk 1]
```diff
@@ -401,4 +403,4 @@ def inject_picker(self, reason: str, page=None) -> bool:
             return True
         except Exception:
             log.exception("Error injecting picker.")
-            return False
+            raise IOError("Error injecting picker.")
```
Comment (fabioz) [inline robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py:406]: This shouldn't raise an exception at this point. Is there a specific reason why you did this change?

Comment (custompointofview) [reply robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py:406]: I haven't changed any PY files today. 
This must have been coming in from `master`.

File: robocorp-code/src/robocorp_code/inspector/web/resources/inspector.css [hunk 2]
```diff
@@ -62,7 +62,7 @@
 }
 
 #inspector-frame.picker::after {
-  content: 'Click element to automate it | Press Shift+ESC to pause picking';
+  content: 'Click element to automate it | Press (Cmd/Win+Shift+ESC) to pause picking';
```
[the remark this answers is not in the document]
Comment (custompointofview) [reply robocorp-code/src/robocorp_code/inspector/web/resources/inspector.css:65]: Sure. I'll add the suggestion. The `Meta` key is the generic name for it.

File: robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py [hunk 3]
```diff
@@ -141,6 +141,7 @@ def page(self, auto_create) -> Optional[Page]:
         self.loop()
 
         page = self._page
+        log.debug("Page is: {page}")
```
Comment (fabioz) [inline robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py:144]: This should probably not be here (I did remove this message, but it's back in this PR).

**Pull request merged**